x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -280,66 +286,3 @@ export const getEndpointAuthzInitialState = (): EndpointAuthz => {
     canReadEventFilters: false,
   };
 };
-
-const SIEM_PERMISSIONS = [
```

Comment:
`canWriteSecuritySolution` and `canReadSecuritySolution` will now come from `fleetAuthz` since we needed access to these within fleet.